app/services/physical_holdings_markup_builder.rb

```diff
@@ -300,7 +300,8 @@ def self.request_label(location_rules)
       'Reading Room Request'
     elsif open_libraries.include?(location_rules['library']['code']) && location_rules["circulates"] == true
       'Request Pick-up or Digitization'
-    elsif open_libraries.include?(location_rules['library']['code'])
+    elsif open_libraries.include?(location_rules['library']['code']) &&
+          location_rules["code"] !="scsbcul" # TODO remove this when columbia is circulating again 
```

Comment:
@carolyncole can you open a new issue and include this line of code and also keep it here?

Reply:
@christinach do you mean open a ticket to remove the line and add it to the other Columbia tickets?  Or is there some reason you want the code in a ticket?

Reply:
I want the TODO reminder in a ticket, but no worries I can open one and add it.

Reply:
because I will forget

Reply:
@christinach I was just clarifying what you wanted in the ticket.  It will be easier to link to once the PR is merged.

Reply:
yes @carolyncole this is what I meant. I should have been more specific in my comment.

Reply:
@christinach https://github.com/pulibrary/orangelight/issues/2220